css/style.css

```diff
@@ -0,0 +1,147 @@
+/* style.css */
+
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+
+.hero {
+  background-image: url(../images/mountain2.jpg);
+  background-size: cover;
+  color: white;
+  height: calc(100vh - 50px);
+
+
+
+/* Hey future Emileigh: This is flex box. it lets you center things and is amazing!*/
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+
+}
+
+.block {
+
+  background-color: black;
+  padding: 40px;
+
+}
+
+
+
+
+a {
+  color: black;
+  font-size: 40px;
+  text-decoration: none;
+  font-family: 'Stalemate', 'Helvetica';
+
+}
+
+h1 {
+  font-size: 50px;
+  font-weight: 400;
+  font-family: 'Raleway', 'Helvetica';
+  /*position: absolute;
+   top: 100px;
+   left: 0;
+   width: 100%;*/
+}
+
+
+h2 {
+  font-size: 70px;
+  font-weight: 400;
+  font-family: 'Stalemate', 'Helvetica';
+  color: #989898;
+  /*position: absolute;
+   top: 150px;
+   left: 0;
+   width: 100%;*/
+
+}
+
+#header {
+  position: relative;
+  width: 100%;
+  text-align: center;
+}
+
+.image {
+  width: 100%;
+  opacity: 0.4;
+  filter: alpha(opacity=40);
```

Comment:
this isn't bad, inherently, you probably just don't need it-- this is how some older browsers (cough cough, IE) support opacity, and from IE 9 on just the `opacity: 0.4` will work. so many other things we're using are broken in IE < 9 that I wouldn't worry about adding this line -- not that it hurts anything by being there. Just good to be aware of where things stand currently.